camelot/admin/entity_admin.py

```diff
@@ -314,17 +314,14 @@ def get_descriptor_field_attributes(self, field_name):
                     # the 'appropriate' way to get it from the class.  Getting it
                     # from the descriptor seems to manipulate  the actual descriptor
                     class_attribute = getattr(self.entity, field_name)
-                    # class attribute of hybrid properties is changed from
-                     # expression to comparator from sqla v1.2 onwards.
-                    if sqlalchemy_version.startswith('1.2') or sqlalchemy_version.startswith('1.3'):
-                        if class_attribute.comparator and isinstance(class_attribute.comparator, hybrid.Comparator):
-                            class_attribute = class_attribute.comparator.expression
+                    # Attribute should always have an expression because of the check made above.
+                    expression = class_attribute.expression
                     if class_attribute is not None:
```

Comment:
de check op class_attribute is not None mt denk ik 1 lijntje hoger

Reply:
Fixed in https://github.com/v-finance/camelot/pull/389.